cms/envs/common.py

```diff
@@ -950,6 +950,9 @@
 
     # management of user-triggered async tasks (course import/export, etc.)
     'user_tasks',
+
+    # Persisting failed tasks, so they can be retried
+    'openedx.core.djangoapps.persistenttask',
```

Comment:
I was getting errors in the tests when it wan't in CMS.  I think I could also add an lms-only decorator to the tests to work around that.